app/src/test/java/org/simple/clinic/home/overdue/phonemask/PhoneMaskBottomSheetControllerTest.kt

```diff
@@ -198,4 +203,45 @@ class PhoneMaskBottomSheetControllerTest {
           listOf(SecureCallClicked, DENIED, Manual),
           listOf(SecureCallClicked, NEVER_ASK_AGAIN, Manual)
       )
+
+  @Test
+  fun `when the screen is created, the patient details must be populated`() {
+    // given
+    val patientSubject = PublishSubject.create<Optional<Patient>>()
+    val patientPhoneNumberSubject = PublishSubject.create<Optional<PatientPhoneNumber>>()
+    whenever(patientRepository.patient(patientUuid)).thenReturn(patientSubject)
+    whenever(patientRepository.phoneNumber(patientUuid)).thenReturn(patientPhoneNumberSubject)
```

Comment:
Both the methods `patientRepository.patient(patientUuid)` and `patientRepository.phoneNumber(patientUuid)` return an `Optional`, this means that either one of them could give out a `None`.
Can we add test to tackle that as well?

Reply:
Also what should be the expected behavior in that case?

Reply:
In this screen, it is not possible for this to happen during normal flows because overdue appointments will never load when the patient and phone number do not exist. I'll change this `setupView` function to use `unwrapJust` instead of `filterAndUnwrapJust` because this is an abnormal state and the user should not be allowed to continue.